AdyenActions/Components/QRCode/ExpirationTimer.swift

```diff
@@ -29,21 +27,23 @@ internal final class ExpirationTimer {
     ///   - onExpiration: The closure called on timer expiration.
     internal init(
         expirationTimeout: TimeInterval,
-        tickInterval: TimeInterval = 1.0,
         onTick: @escaping (TimeInterval) -> Void,
         onExpiration: @escaping () -> Void
     ) {
         self.expirationTimeout = expirationTimeout
-        self.tickInterval = tickInterval
```

Comment:
Wouldn't removing this limit flexibilty and testing?

Reply:
I removed it as it was unused and also implied to me that we could speed up the timer, but what it did instead was just notify more often 

but **not** speed up or slow down the timer.

Reply:
As it uses TimeInterval (which implies a duration of seconds) the only valid usecase of this would be for testing but as it was not used so far I removed it. Better to add it later than adding tests for an unused feature imo.
I agree tho that it could be used to speed up tests in the future.